liquibase-standard/src/main/java/liquibase/integration/spring/SpringLiquibase.java

```diff
@@ -254,6 +254,7 @@ public void afterPropertiesSet() throws LiquibaseException {
                             final Connection c = getDataSource().getConnection();
                             liquibase = createLiquibase(c);
                             generateRollbackFile(liquibase);
+                            liquibase.setFast_check_disabled(Boolean.TRUE);
```

Comment:
Should we change the default? This might cause a *small* performance hit, right?